plugins/woocommerce/composer.json

```
@@ -13,19 +13,20 @@
 		}
 	],
 	"require": {
-		"php": ">=7.0",
+		"php": ">=7.2",
 		"automattic/jetpack-autoloader": "2.10.1",
 		"automattic/jetpack-constants": "1.5.1",
-		"composer/installers": "~1.7",
-		"maxmind-db/reader": "1.6.0",
-		"pelago/emogrifier": "3.1.0",
+		"composer/installers": "^1.9",
+		"maxmind-db/reader": "^1.6",
```

Regarding the installed version, at time of writing it will be 1.11.0 (which shares the [same min PHP requirements](https://github.com/maxmind/MaxMind-DB-Reader-php/blob/v1.11.0/composer.json#L16)). Worth bumping here, too?

```suggestion
		"maxmind-db/reader": "^1.11",
```

(Not essential, just a thought.)

---

Makes sense, changed.